app/components/publications/publications-table-row.hbs

```diff
@@ -158,48 +210,52 @@ request the whole row and not only the arrow at the end to be clickable }}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.publicationReceivedDate}}
-    <td class="auk-u-text-nowrap">
-      {{#if @publicationFlow.publicationSubcase.receivedDate}}
-        {{moment-format
-          @publicationFlow.publicationSubcase.receivedDate
-          "DD-MM-YYYY"
-        }}
-      {{else}}
-        {{t "dash"}}
-      {{/if}}
-    </td>
-  {{/if}}
-  {{#if @tableColumnDisplayOptions.translationReceivedDate}}
-    <td class="auk-u-text-nowrap">
-      {{#if @publicationFlow.translationSubcase.receivedDate}}
-        {{moment-format
-          @publicationFlow.translationSubcase.receivedDate
-          "DD-MM-YYYY"
-        }}
+  {{#if @tableColumnDisplayOptions.publicationDueDate}}
+    <td
+      data-test-publication-table-row-publication-due-date
+      class="auk-u-text-nowrap"
+    >
+      {{#if @publicationFlow.publicationSubcase.dueDate}}
+        {{!Ember seems not to await async getter}}
+        <div class="{{if (await this.isPublicationToLate) "auk-u-text-bold"}}">
```

Comment:
Ik denk dat je deze `await`s kan vermijden door de functies die je nu in `publication-utils.js` hebt gedefineerd op de Ember Data models toe te voegen.

Vb. op publication-subcase
```javascript
get isTooLate() {
    return moment(this.dueDate).isBefore(Date.now(), 'day')
}
```

Op die manier kan je in de template gewoon `@publicationFlow.publicationSubcase.isTooLate` gebruiken.

Gelijkaardige redenering voor de `isClosed` op publication-status.

Reply:
Waarom ik het niet zo gedaan heb is omdat dit logica is die businesskennis vereist. Die kan volgens mij best maar op 1 plaats uitgeschreven staan. Als ik de logica in de models stak, stak de kennis deels in `publication-subcase`, deels in `publication-status` en deels in de template.
Om die awaits te vermijden, stel ik `@tracked`-properties voor.

Reply:
`@tracked`-properties zijn ook goed voor mij.

Reply:
Nu ik er nog eens naar kijk, lijkt de 'businesskennis' die nodig is mij hier eerder verwaarloosbaar en is het eerder logisch om de derived properties toe te voegen aan de ember-data models. 

- Voor publication-status is er intussen in de development branch vanuit een PR een derived property `isFinished` toegevoegd die je kan gebruiken.
- Voor de translation-subcase en publication-subcase modellen met een attribute `dueDate` lijkt een derived property `isOverdue` niet geheel onlogisch.